addons/common/XEH_postInit.sqf

```diff
@@ -55,6 +56,23 @@
     _vehicle setVariable [QGVAR(blockEngine), _set > 0, true];
     _vehicle engineOn false;
 }] call CBA_fnc_addEventHandler;
+[QGVAR(blockThrow), {
+    params ["_object", "_set"];
+    if (_set > 0) then {
+        GVAR(blockThrow_EH) = ["KeyDown", {(_this select 1) in actionKeys "Throw"}] call CBA_fnc_addDisplayHandler;
```

Comment:
Does this work if you bind it to a mouse (or a controller/joystick)?

Reply:
no naturally